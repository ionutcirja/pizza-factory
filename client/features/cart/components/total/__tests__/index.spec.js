import React from 'react';
import { shallow } from 'enzyme';
import { Total } from '..';

describe('Total component', () => {
  const propsToRender = {
    price: '25.7',
    theme: {
      colours: {
        darkBlue: 'dark blue',
      },
    },
  };
  
  describe('render', () => {
    it('should render the total price', () => {
      const wrapper = shallow(<Total {...propsToRender} />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
