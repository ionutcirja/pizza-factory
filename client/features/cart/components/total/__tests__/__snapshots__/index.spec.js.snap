// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List component render should render the total price 1`] = `
<TotalWrapper
  colour="dark blue"
>
  Total: 25.7
</TotalWrapper>
`;
